src/slurm/Makefile.am

```diff
@@ -0,0 +1,23 @@
+
+plugindir = $(libdir)/slurm
```

Comment:
This plugindir seems to be dependent on the --prefix or --libdir passed to Singularity's build which could in theory clash with Slurm's own library directory. While this may not cause any specific problems, I would suggest staying out of the Slurm internal library/plugin directory. Additionally, this will have to be added to the Slurm plugin include file anyway. How about `plugindir = $(libdir)/singularity/slurm` instead?